webapp/views/App/views/Users/UserEdit/UserEdit.js

```diff
@@ -38,79 +39,95 @@ const UserEdit = () => {
   const i18n = useI18n()
   const validation = User.getValidation(userToUpdate)
 
+  const titleItems = ['mr', 'ms', 'preferNotToSay']
+  const onUpdateTitle = (value) => {
+    const currentProps = User.getProps(userToUpdate)
+    return onUpdate({ name: User.keys.props, value: User.assocPropsTitle(value)(currentProps) })
```

Comment:
User.assocPropsTitle(value)(currentProps) I was expecting that this function returns a user with the props but it returns the props, how about:

UserProps. assocPropsTitle
or 
User.Props.assocTitle
or
User.propsAssocTitle

not really sure

Reply:
Why we don't have typescript >:)

Reply:
Maybe in this case `User.Props.assocTitle`?